zmsentities/tests/Zmsentities/ExchangeTest.php

```
@@ -114,6 +114,73 @@ public function testGroupedWithRequestsSum()
+            $result-data['average_processingtime_overall']
+        );
+    }
+
+    public function testwithWeightedAverageProcessingTimeIncludesUncategorized()
+    {
+        $exchange = new \BO\Zmsentities\Exchange();
+
+        $exchange->data = [
+            'sum' => [
+                'Personalausweis' => 2,
+                \BO\Zmsentities\Exchange::REQUEST_STAT_NAME_UNCATGORIZED => 1,
```

PHPStan (2.2.8)

[error] Access to undefined constant BO\Zmsentities\Exchange::REQUEST_STAT_NAME_UNCATGORIZED.


(classConstant.notFound)